gnd/src/main/java/com/google/android/gnd/ui/map/MapProvider.java

```diff
@@ -62,5 +60,7 @@
     void enableCurrentLocationIndicator();
 
     void updateMarkers(ImmutableSet<Feature> features);
+
+    void renderJsonLayer();
```

Comment:
Now that we're switching to "productionize" phase, please be sure to add Javadoc for all new public methods like this one.

Reply:
Actually did you mean to add this now? I think for the adapter we'd want to have something that would allow the caller to add/remove/show/hide JSON layers, but the usage will probably only be clear once we implement the UI. Can we remove this interface method for now?

Reply:
You're totally right! We don't need it. I just did a plain old revert to undo the changes, so I guess this method was already in prior to the commit I reverted. I've removed it.